config/src/config/safety_rules_config.rs

```diff
@@ -36,7 +37,8 @@ impl Default for SafetyRulesConfig {
             service: SafetyRulesService::Thread,
             test: None,
             verify_vote_proposal_signature: true,
-            // Default value of 30seconds for a timeout
+            export_consensus_key: false,
```

Comment:
hmm....

@mgorven, is this good for you?

Reply:
Defaulting to remote signing? Fine with me!

Reply:
Sounds good! I'll push it :smile